local_planner/src/local_planner.cpp

```diff
@@ -428,9 +417,9 @@ void LocalPlanner::getStateAndTwistInput() {
   ref_body_plan_(0, 11) = cmd_vel_[5];
 
   // Only adaptive pitch
-  ref_body_plan_(0, 4) = local_footstep_planner_->getTerrainSlope(
-      current_state_(0), current_state_(1), current_state_(6),
-      current_state_(7));
+  // ref_body_plan_(0, 4) = local_footstep_planner_->getTerrainSlope(
```

Comment:
remove?

Reply:
I want to keep them. There are two ways to align with terrain, here is another way.

Reply:
Ok, maybe just add a comment to clarify that